tsdb/index/tsi1/index.go

```diff
@@ -802,21 +802,11 @@ func (i *Index) compactionGroups(fileSet FileSet) [][]*IndexFile {
 
 		// Skip over log files. They compact themselves.
 		if !ok {
-			if isCompactableGroup(group, i.CompactionFactor) {
-				group, groups = nil, append(groups, group)
-			} else {
-				group = nil
-			}
 			continue
 		}
 
 		// If file is currently compacting then stop current group.
 		if indexFile.Compacting() {
-			if isCompactableGroup(group, i.CompactionFactor) {
-				group, groups = nil, append(groups, group)
-			} else {
-				group = nil
-			}
 			continue
 		}
```

Comment:
This section is needed because we're looking for contiguous runs of non-compacting files. If we hit a file that is compacting then we need to make a compaction group with the previous files, if applicable.